src/components/ae-panel/ae-panel.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="ae-panel">
+    <slot />
+  </div>
+</template>
+<script>
+export default {
+  name: 'ae-panel',
+};
+</script>
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+
+.ae-panel {
+  position: relative;
```

Comment:
`relative` positioning should be enabled in special cases, I don't see reasons for it here.

Reply:
Its just a default behaviour I would like the component to have! It makes sense in case you want to position elements relative to the panel, which in the future might happen.

Reply:
This is the reason why I won't use this component.